shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -1449,6 +1449,14 @@ public void removeFlutterEngineAttachmentListener(
         .send();
   }
 
+  private FlutterViewDelegate delegate = new FlutterViewDelegate();
+
+  /** Set the FlutterViewDelegate, such as to a mock for testing. */
```

Comment:
nit: I think this comment can be removed (it setting is intuitive, it being used for testing is implied by `@VisibleForTesting`). If you wanted to keep it, I'd be more explicit ("... only used for testing").